src/App.tsx

```diff
@@ -1,10 +1,54 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { Content } from './components/Content/Content';
+import { Notification } from './components/Notification/Notification';
 
-const USER_ID = 0;
+const USER_ID = 10888;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isAnyError, setIsAnyError] = useState(false);
+  const [isErrorMessage, setIsErrorMessage] = useState(false);
+  const [filterCondition, setFilterCondition] = useState('all');
```

Comment:
can we create a const or enum with our statuses and use it in all places in our application when we need status?